engine/apps/slack/scenarios/slack_channel_integration.py

```diff
@@ -82,46 +82,32 @@ def save_thread_message_for_resolution_note(
         if result["permalink"] is not None:
             permalink = result["permalink"]
 
-        try:
-            slack_thread_message = ResolutionNoteSlackMessage.objects.get(
-                ts=message_ts,
-                thread_ts=thread_ts,
-                alert_group=alert_group,
-            )
-            if len(text) > 2900:
-                if slack_thread_message.added_to_resolution_note:
-                    self._slack_client.api_call(
-                        "chat.postEphemeral",
-                        channel=channel,
-                        user=slack_user_identity.slack_id,
-                        text=":warning: Unable to update the <{}|message> in Resolution Note: the message is too long ({}). "
-                        "Max length - 2900 symbols.".format(permalink, len(text)),
-                    )
-                return
-            slack_thread_message.text = text
-            slack_thread_message.save()
+        slack_thread_message, created = ResolutionNoteSlackMessage.objects.get_or_create(
```

Comment:
I think it's better to check `len(text)` before creating ResolutionNoteSlackMessage. Too long text may cause issues either with saving it to the db or displaying it in Slack 🤔

Reply:
👍 

do you happen to know why there is this condition?:
```python3
if slack_thread_message.added_to_resolution_note:
```
if that can be removed, we can still use `get_or_create`

Reply:
alternative solution is to keep the separate `get`/`create` calls as it was, and add _another_ `try/except` when creating, and catch any possible `IntegrityError`s (but this seems a bit messier imo)